src/js/actions/AppActions.js

```diff
@@ -31,23 +31,22 @@ const init = () => {
         }
     });
 
-    Player.getAudio().addEventListener('play', () => {
+    Player.getAudio().addEventListener('play', async () => {
```

Comment:
Why would we use async here ? Why not use it everywhere ? 

More generally, is async really meant to replace ALL asynchronous stuff, I'm not sure.